### ISSUE TITLE
No "stylelint" targets found.

### ISSUE THREAD

**Issue by jitendravyas** [Dreamseer/stylelint-formatter-pretty#2]
```
❯ grunt stylelint
Loading "Gruntfile.js" tasks...ERROR
>> TypeError: Cannot read property 'sort' of undefined
>> No "stylelint" targets found.
Warning: Task "stylelint" failed. Use --force to continue.

```

```
		stylelint: {
			    options: {
      formatter: require('stylelint-formatter-pretty')()
    },
			all: ['app/styles/**/*.scss']
		},
```

**Comment by Dreamseer** [comment]
Hi @jitendravyas, thanks for your bug report.

Does this warning occur when you remove the formatter, too? It seems like Stylelint could not find your files in `'app/styles/**/*.scss'`.

Anyway I will add a test for this case and prevent the `TypeError`.
